src/lib/tracer.ts

```diff
@@ -0,0 +1,67 @@
+import config from "config"
+import { verbose as debug, error } from "./loggers"
+import tracer from "dd-trace"
+import { Tags } from "opentracing"
+
+const { DD_TRACER_SERVICE_NAME, DD_TRACER_HOSTNAME, PRODUCTION_ENV } = config
+
+export function init() {
+  tracer.init({
+    service: DD_TRACER_SERVICE_NAME,
+    hostname: DD_TRACER_HOSTNAME,
+    plugins: false,
+    logger: { debug, error },
+    debug: !PRODUCTION_ENV,
+  })
+  tracer.use("express", {
+    service: `${DD_TRACER_SERVICE_NAME}.request`,
+  })
+  tracer.use("http", {
+    service: `${DD_TRACER_SERVICE_NAME}.http-client`,
+  })
+  tracer.use("graphql", {
+    service: `${DD_TRACER_SERVICE_NAME}.graphql`,
+    filterVariables: variables => variables,
```

Comment:
This means we capture all variables. When/if needed, we can redact some sensitive ones here.

Reply:
Maybe move your comment into the code so that its clearer for other devs?